src/appleseed/renderer/modeling/project/projectfilewriter.cpp

```diff
@@ -929,9 +928,9 @@ bool ProjectFileWriter::write_project_file(
 }
 
 bool ProjectFileWriter::write(
-        const Project&  project,
-        const char*     filepath,
-        const int       options)
+  const Project&  project,
```

Comment:
Please follow the indentation style of the other functions: arguments are indented normally (4 spaces).